src/env.h

```diff
@@ -666,7 +670,13 @@ class Environment {
   // List of id's that have been destroyed and need the destroy() cb called.
   inline std::vector<double>* destroy_async_id_list();
 
-  std::unordered_multimap<int, loader::ModuleWrap*> module_map;
+  std::unordered_multimap<int, loader::ModuleWrap*> hash_to_module_map;
+  std::unordered_multimap<uint32_t, loader::ModuleWrap*> id_to_module_map;
+  std::unordered_multimap<uint32_t, contextify::ContextifyScript*>
```

Comment:
This seems like a lot of maps to me - are you sure this is the simplest way to do this?

Reply:
it's a lot of maps. it's also the only way. We've talked to V8 about this before, but we're never going to be able to identify modules except by js primitive values so I use number -> wrap maps.

Reply:
Why is this a multimap? Shouldn't the ID uniquely identify the module?